custom-words.txt

```diff
@@ -587,6 +588,7 @@ Dsms
 DTDL
 Dtus
 duration'PT
+durring
```

Comment:
typo? 
you have to change upstream DSMainDev to fix the description typo

Reply:
It is a typo in DSMainDev. I have fixed it there but it is not propagated to the swagger creation(That it is because those errors go through nugets which need a nuget bump. Since that is a really long process, I have fixed it in DsMainDev and removed it from custom-words. We can fix this in next API change if it is ok(When the error fix gets to nugets)

Reply:
OK. we can wait for the nuget update